src/content/quickmove.js

```diff
@@ -253,16 +255,21 @@ var quickmove = (function() {
        * to iterate through any sub-sub-folders.
        *
        * @param aFolder  the folder to check
+       * @param excludeArchives  if Archives folder must be excluded
        */
-      function processFolder(aFolder) {
+      function processFolder(aFolder, excludeArchives) {
+         if (excludeArchives 
+          && aFolder.isSpecialFolder(0x00004000, false)) {
```

Comment:
Instead of the number I think you can use `Ci.nsMsgFolderFlags.Archive`

Reply:
```suggestion
          && aFolder.isSpecialFolder(Ci.nsMsgFolderFlags.Archive, false)) {
```

Reply:
OK, I have updated with the constant. Thank you.